### PR TITLE
fix: make responsive forms of activities section #537

### LINKED ISSUE #537: Fix forms of activities section
Subtask of #529

### PR DESCRIPTION
**Pull request by Angeluz-07**
close #537

### CODE REVIEW

File: src/app/modules/activities-management/pages/activities-management.component.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.activity-container{
+    width: 100%;
```
Comment (juanultimate) [inline src/app/modules/activities-management/pages/activities-management.component.scss:2]: bootstrap has a w-100 class which does this

Comment (Angeluz-07) [reply src/app/modules/activities-management/pages/activities-management.component.scss:2]: Yes bootstrap has those classes, but I created this class to use the media-query. On desktop screen we need 80% of width, but in phone we need 100%. Kind of style conditionally.

Comment (Angeluz-07) [reply src/app/modules/activities-management/pages/activities-management.component.scss:2]: I am not sure if can do it, only with bootstrap classes, without using empty divs.

Comment (Angeluz-07) [reply src/app/modules/activities-management/pages/activities-management.component.scss]: You gave me the idea. I could do it with bootstrap classes only. Thanks

**Pull request merged**